bugshaker/src/main/res/layout/activity_main.xml

```diff
@@ -209,6 +216,15 @@
         android:layout_marginTop="3dp"
         android:layout_weight="1"></com.github.stkent.bugshaker.flow.widget.DrawingView>
 
+    <EditText
+        android:id = "@+id/textEditing"
```

Comment:
We use this EditText; it's just that we shouldn't be creating an EditText in our code so we got rid of that code